tests/snowflake/operators/test_snowflake.py

```diff
@@ -32,49 +32,13 @@
 SINGLE_STMT = "select i from user_test order by i;"
 
 
-def test__check_queries_finish_success():
-    mock_conn = MagicMock()
-    mock_conn.get_query_status.return_value = QueryStatus.SUCCESS
-    assert _check_queries_finish(mock_conn, ["test_sfqid_1", "test_sfquid_2"]) is True
-
-
 @pytest.mark.parametrize(
-    "mock_status",
-    (
-        QueryStatus.RUNNING,
-        QueryStatus.QUEUED,
-        QueryStatus.RESUMING_WAREHOUSE,
-        QueryStatus.BLOCKED,
-        QueryStatus.NO_DATA,
-        QueryStatus.FAILED_WITH_INCIDENT,
-    ),
+    "query_status,expected_response", [(QueryStatus.SUCCESS, True), (QueryStatus.RUNNING, False)]
```

Comment:
I think we should still check other status also return False

Reply:
Since we do not handle error cases here so not sure if a test requires. quick exploration says they have some tests in https://github.com/snowflakedb/snowflake-connector-python/blob/9b586d4c37f5206742f418ae1d66fc83248cacbe/test/unit/test_connection.py

Reply:
It would still be better if we could verify those value returns `False`. But I can go with the current implementation as well